custom-metrics-stackdriver-adapter/examples/direct-to-sd/README.md

```diff
@@ -8,8 +8,13 @@ a [custom metrics setup] in your cluster.
 
 A simple sd-dummy-exporter container exports a metric of constant value 
 to Stackdriver in a loop. The metric name and value can be passed in via flags.
-Pod id is passed to the container via downward API (see [custom-metrics-sd deployment]
-for how it's done)
+Pod id and pod name is passed to the container via downward API (see
+[custom-metrics-sd deployment] for how it's done).
+
+Stackdriver dummy exporter can export metrics for **new Stackdriver resource model**,
+**legacy Stackdriver resource model** or both, specified by flags:
+`--use-new-resource-model` and `--use-old-resource-model`. The default
+deployment exports metrics for both resource models at the same time.
```

Comment:
What is a default deployment now?

Reply:
The one used in this README - clarified.